selenium/src/test/java/au/org/emii/portal/tests/PointTimeSeriesLatLonTest.java

```diff
@@ -46,17 +48,28 @@ public void validateLatLon(int x, int y) {
         getDriver().switchTo().activeElement();
 
         String popupHtml = webElementUtil.waitForElement(By.xpath("//div[contains(@class,'popupHtml') and contains(.,'Lat:')]")).getText();
+        String[] parts = popupHtml.split(" ");
+
+
         getDriver().switchTo().window(mainWindowHandle);
 
-        WebElement latitude2 = webElementUtil.findElement(By.xpath("//label[text()='Latitude']/following-sibling::input"));
-        Assert.assertNotNull(latitude2);
-        String lat = latitude2.getAttribute("value");
-        Assert.assertTrue(popupHtml.contains(lat));
+        WebElement latitudeTextBox = webElementUtil.findElement(By.xpath("//label[text()='Latitude']/following-sibling::input"));
+        Assert.assertNotNull(latitudeTextBox);
+        double lat =  Double.parseDouble(latitudeTextBox.getAttribute("value"));
+        double featureInfoLatitude = Double.parseDouble(parts[1]);
+        double latDifference = lat - featureInfoLatitude;
```

Comment:
Does this difference need to be absolute?

Reply:
No, because it was checking for both negative and positive difference:
```latDifference > -0.1 && latDifference < 0.1```

But converting it to absolute is way cleaner so I've added a commit to do that instead.